### PR TITLE
Feature/#3259 view of the pricing page for a super admin 

### PR DESCRIPTION
**Pull request by nazarf1**
created pricing page for super admin according to mock ups
logic will be added later

### CODE REVIEW

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+.main-content {
+  margin: 15px 0 0 35px;
+}
+
+h4 {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 32px;
+  letter-spacing: 0.002em;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss:15]: image
Its less then 1px

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  input {
+    width: 100px;
+  }
+}
+
+.input4 {
+  input {
+    width: 123px;
+  }
+}
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss:101]: Can we make more appropriate names for classes? 😸

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss (hunk 1 above)
Comment (nazarf1) [reply src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss]: done

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+.buttons {
+  margin-right: 45px;
+  margin-top: 55px;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss:105]: We can combine this into one:
margin: 55px 45px 0 0;

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-delete-popup/ubs-admin-tariffs-delete-popup.component.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+@Component({
+  selector: 'app-ubs-admin-tariffs-delete-popup',
+  templateUrl: './ubs-admin-tariffs-delete-popup.component.html',
+  styleUrls: ['./ubs-admin-tariffs-delete-popup.component.scss']
+})
+export class UbsAdminTariffsDeletePopupComponent implements OnInit {
+  constructor() {}
+
+  ngOnInit(): void {}
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-delete-popup/ubs-admin-tariffs-delete-popup.component.ts:11]: empty, remove)

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.scss [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+  }
+
+  ::placeholder {
+    padding: 5px 0 0 5px;
+  }
+}
+
+.submit-btn {
+  margin-top: 20px;
+  margin-bottom: 20px;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.scss:210]: margin: 20px 0;

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss (hunk 2 above)
Comment (nazarf1) [reply src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss]: done

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.scss [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+}
+
+img {
+  cursor: pointer;
+}
+
+.image-buttons {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 7px 0 7px;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.scss:220]: let it be
But it's better to use shorter version in the future:
padding: 0 7px;

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.ts [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  styleUrls: ['./ubs-admin-tariffs.component.scss']
+})
+export class UbsAdminTariffsComponent implements OnInit {
+  public icons = {
+    edit: './assets/img/profile/icons/edit.svg',
+    delete: './assets/img/profile/icons/delete.svg'
+  };
+  constructor(public dialog: MatDialog) {}
+  ngOnInit(): void {}
+
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.ts:18]: empty hook

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss (hunk 3 above)
Comment (nazarf1) [reply src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component.scss]: done

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.scss (hunk 5 above)
Comment (nazarf1) [reply src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.scss]: done

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.scss (hunk 6 above)
Comment (nazarf1) [reply src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.scss]: done

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.ts (hunk 7 above)
Comment (nazarf1) [reply src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.ts:18]: should i delete it if in the near future this hook won't be empty?

Comment (Edw1x) [reply src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.ts:18]: Yes, you will add it in the future))

Comment (nazarf1) [reply src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.ts]: done)

File: src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-delete-popup/ubs-admin-tariffs-delete-popup.component.ts (hunk 4 above)
Comment (nazarf1) [reply src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-delete-popup/ubs-admin-tariffs-delete-popup.component.ts]: done

**Review by Edw1x — APPROVED** [review]

**Review by NewArtMask — APPROVED** [review]

**Review by nazarhlyvka — APPROVED** [review]

**Review by KhrystynaPr — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Pull request merged**